src/DataSource.ts

```diff
@@ -43,39 +36,6 @@ export class GitHubDataSource extends DataSourceWithBackend<GitHubQuery, GitHubD
     return replaceVariables(this.templateSrv, query, scoped);
   }
 
-  /**
-   * Implemented as part of the DataSourceAPI. It allows the datasource to serve as a source of annotations for a dashboard.
-   * @returns A promise that resolves to an array of AnnotationEvent objects representing the annotations for the dashboard.
-   * @todo This is deprecated and it is recommended to use the `AnnotationSupport` feature for annotations.
-   */
-  async annotationQuery(request: any): Promise<AnnotationEvent[]> {
```

Comment:
I know this was deprecated but do we have any other way to support annotations?

Reply:
I think that's doing querying too but the new approach is doing that already out of the box. So, all good. Thank you!